Dockerfile

```diff
@@ -1,35 +1,50 @@
 FROM node:7.9.0
 
-# Note: npm is v2.15.11
-RUN \
-  apt-get install tar bzip2 && \
-  npm install -g ember-cli@2.13.3 && \
-  npm install -g bower@1.8.0 && \
-  echo '{ "allow_root": true }' > /root/.bowerrc && \
-  npm install -g phantomjs-prebuilt@2.1.14 && \
-  # install watchman
-  # Note: See the README.md to find out how to increase the
-  # fs.inotify.max_user_watches value so that watchman will
-  # work better with ember projects.
-  git clone https://github.com/facebook/watchman.git && \
-  cd watchman && \
-  git checkout v3.5.0 && \
-  ./autogen.sh && \
-  ./configure && \
-  make && \
-  make install
+MAINTAINER Preston Sego
+MAINTAINER Akram Mnif
 
-# Ensures that the installed executables are in the path
-ENV PATH "$PATH:/usr/local/lib/node_modules"
+USER root
+
+ENV PATH $PATH:/usr/local/lib/node_modules
+
+RUN `# Create web directory`                                                                             && \
+     mkdir /web                                                                                          && \
+                                                                                                            \
+    `# Install Google Chrome Stable version`                                                             && \
+     wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add -                  && \
+     echo "deb http://dl.google.com/linux/chrome/deb/ stable main" > /etc/apt/sources.list.d/google.list && \
+     apt-get update                                                                                      && \
+     apt-get install -y google-chrome-stable                                                             && \
```

Comment:
I think at this point, It'd be better to have the && at the beginning of the line.
the amount of white space below is ridiculous. lol